tests/components/devialet/test_init.py

```diff
@@ -13,18 +17,19 @@ async def test_load_unload_config_entry(
     hass: HomeAssistant, aioclient_mock: AiohttpClientMocker
 ) -> None:
     """Test the Devialet configuration entry loading and unloading."""
-    entry = await setup_integration(hass, aioclient_mock)
+    with patch.object(DevialetApi, "upnp_available", return_value=True):
```

Comment:
```suggestion
    with patch("homeassistant.components.devialet.DevialetApi.upnp_available", return_value=True):
```

The integration would be a lot nicer if we could patch the library instead of the HTTP calls

Reply:
Why do you prefer the patch above the patch.object here?